README.md

```diff
@@ -100,7 +100,12 @@ A glossary entry is structured like this:
         and the value may contain local links to other terms in this glossary
         (i.e., links starting with `#`)
         and/or links to outside sources.
+### Building a local test site
+Once you have finished creating your glossary entry it may be useful to build a local test site to check the syntax and formatting is correct. To do this you will need a working installation of Jekyll on your local machine (instructions for which can be found [here](https://jekyllrb.com/docs/installation/).
```

Comment:
```suggestion
### Building a local test site (optional)
*N.B. this is not a required step for submitting a PR to Glosario.*

Once you have finished creating your glossary entry it may be useful to build a local test site to check the syntax and formatting is correct. To do this you will need a working installation of Jekyll on your local machine (instructions for which can be found [here](https://jekyllrb.com/docs/installation/).
```

@bjthorpe 

I understand the intention here, and I think providing a bit more information about how one might test the site is good. However, I'm concerned that this wording might make people feel as if they are expected to test the deployment before they submit a PR. We have been trying to make this project more inclusive and welcoming to people who may not know a lot about GitHub, local deployment, or other things. The submission of new entries offers a way to begin contributing to open-source projects that doesn't require a lot of previous experience.

As the person who is generally looking in on PRs to see why builds fail, I can attest that the error messages thrown up by the linter and other things can be quite arcane, and I do not want to place responsibility for that on newcomers. I do, however, let them know why things are failing, so that when they (hopefully) contribute again, they understand how to avoid the same errors.